### PR TITLE
[UI] Display/allow user to copy alternate Epic login URL

### PR DESCRIPTION
**Pull request by kshitijsubedi**
Added Features:
- Displays login URL
- Button to Copy the URL to the clipboard
- Button to Open Link in the Browser

Related Issue: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/2245

Before: 
image

After: 
image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by imLinguin — APPROVED** [review]
Looks good to me

File: src/frontend/screens/Login/components/SIDLogin/index.css [hunk 1]
```diff
@@ -54,6 +54,18 @@
   color: var(--link-highlight);
 }
 
+.loginInstructions .login-link {
+  display: flex;
+  background-color: var(--background-darker);
+  border-color: var(--input-background);
```
Comment (Nocccer) [inline src/frontend/screens/Login/components/SIDLogin/index.css:60]: To make this buttons align with the themes i think it should be `--accent`. The icon and text color should also be `--accent`.

**Review by Nocccer — CHANGES_REQUESTED** [review]
LGTM to beside the button color.

File: src/frontend/screens/Login/components/SIDLogin/index.css (hunk 1 above)
Comment (kshitijsubedi) [reply src/frontend/screens/Login/components/SIDLogin/index.css]: Updated with accent color

**Review by Nocccer — APPROVED** [review]
LGTM. Thanks for contributing

**Pull request merged**